Sources/VGSShowSDK/UIElements/VGSLabel/VGSLabel.swift

```diff
@@ -147,6 +158,25 @@ public final class VGSLabel: UIView, VGSLabelProtocol {
 		copyText(format: format)
 	}
 
+  /// An array of `VGSTextRanges`, where `VGSLabel.secureTextSymbol` should replace text character
+  internal var secureTextRanges: [VGSTextRange]?
+
+  /// Set `VGSLabel.secureTextSymbol` in label text
+  public func setSecureText(start: Int? = nil, end: Int? = nil) {
```

Comment:
Would suggest to rename this function to `setSecureTextRange` to emphasize on face that we specify range.